extensions/binding/org.eclipse.smarthome.binding.dmx.test/OSGI-INF/DmxTestHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please replace this file with the according DS annotations? For an example you can have a look at the [SonosHandlerFactory](https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/internal/SonosHandlerFactory.java#L46) But since this is a "test service" you can leave out the "name=" parameter so it uses the default name